tools/Microsoft.Health.Extensions.BuildTimeCodeGenerator/Microsoft.Health.Extensions.BuildTimeCodeGenerator.nuspec

```diff
@@ -7,13 +7,14 @@
         <owners>Microsoft.Health.Extensions.BuildTimeCodeGenerator</owners>
         <requireLicenseAcceptance>false</requireLicenseAcceptance>
         <description>Package Description</description>
-        <repository type="git" url="https://github.com/microsoft/healthcare-components.git" />
+        <repository type="git" url="https://github.com/microsoft/healthcare-shared-components.git" />
     </metadata>
     <files>
         <file src="_._" target="lib/netstandard2.0/" />
         <file src="Microsoft.Health.Extensions.BuildTimeCodeGenerator.targets" target="build/netstandard2.0/Microsoft.Health.Extensions.BuildTimeCodeGenerator.targets" />
 
         <file src="Microsoft.Health.Extensions.BuildTimeCodeGenerator.targets" target="tools/" />
         <file src="$publishdir$\netcoreapp3.1\**\*" target="tools/netcoreapp3.1/" />
+        <file src="$publishdir$\net5.0\**\*" target="tools/net5.0/" />
```

Comment:
>file [](start = 9, length = 4)

it's gonna be one chunky nuget file :D

Reply:
Yeah, I'm not sure this is quite the right design here. 50MB of runtime (now *2) doesn't seem right. 
Out of scope for this PR perhaps.

Reply:
Yeah, this shouldn't be forever it should just give us a ramp to upgrade our projects and drop 3.1.
In size its ~30mb compressed as a nuget, even with the second version its still smaller than self-contained.

Reply:
I guess I'm surprised that a full publish is needed there.